modules/kms/providers.tf

```diff
@@ -0,0 +1,29 @@
+module "iam_roles" {
```

Comment:
nitpick: the content is the same, but the modules are in a different order than usual in this file. Could we replace this file with the same, latest provider.tf that we've been using in other components? Might help prevent future confusion